playwright-e2e/tests/dsm/osteo/os2-display-verification.spec.ts

```diff
@@ -1011,12 +1011,6 @@ test.describe.serial(`${StudyName.OSTEO2}: Verify expected display of participan
 
     await test.step(`Check the display of profile info`, async () => {
       //Check profile webelements specific to OS2
-      const somaticConsentStatus = participantPage.getSomaticConsentStatusLocator;
```

Comment:
Where have these assertions gone?  Do we no longer need them?

Reply:
nope - it's a weird case where Test and Staging have them (staging has it listed a little differently e.g. as "Somatic Consent / Assent")  but Dev and Prod do not - the ticket for them is actually in the backlog: https://broadworkbench.atlassian.net/browse/PEPPER-1265 - I made the test using OS2 Test, so didn't see the difference at first